specs/electra/p2p-interface.md

```diff
@@ -105,7 +105,7 @@ The following convenience variables are re-defined
 The following validations are added:
 
 - [REJECT] `len(committee_indices) == 1`, where
-  `committee_indices = get_committee_indices(aggregate)`.
+  `committee_indices = get_committee_indices(aggregate.committee_bits)`.
 - [REJECT] `aggregate.data.index == 0`
```

Comment:
Good catch. I find this section to be a little confusing. It's missing the `aggregate = signed_aggregrate_and_proof.message.aggregate` alias. In this PR, could you also add a paragraph near the beginning similar to the following:

https://github.com/ethereum/consensus-specs/blob/11e15092dccdffed0db15a0314682ee0c5737eec/specs/gloas/p2p-interface.md?plain=1#L250-L252